challenges/easy/caesar-cipher/solutions/valentinadagata/main.js

```diff
@@ -0,0 +1,69 @@
+const letters = 26;
+
+function encode(str, shift) {
+  shift  = shift % letters
```

Comment:
Please create a new constant value instead of re-assigning the function parameter 🙂